<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_ImageSave</class>
 <widget class="QDialog" name="Dialog_ImageSave">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save Image</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>YUV</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0" colspan="4">
        <widget class="QGroupBox" name="groupBox_BufferInfo">
         <property name="title">
          <string>Crop Window</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>X:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinBox_BufferP_4">
            <property name="maximum">
             <number>8192</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>X:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QSpinBox" name="spinBox_BufferP_5">
            <property name="maximum">
             <number>8192</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Width</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QSpinBox" name="spinBox_BufferP_3">
            <property name="maximum">
             <number>8192</number>
            </property>
            <property name="value">
             <number>1920</number>
            </property>
           </widget>
          </item>
          <item row="0" column="6">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Height</string>
            </property>
           </widget>
          </item>
          <item row="0" column="7">
           <widget class="QSpinBox" name="spinBox_BufferH_3">
            <property name="maximum">
             <number>8192</number>
            </property>
            <property name="value">
             <number>1080</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Format:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="QComboBox" name="comboBox_YUVFormat">
         <item>
          <property name="text">
           <string>YUV420_IYUV      YYYY YYYY UU VV</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>YUV420_YV12     YYYY YYYY VV UU</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>YUV420_NV12     YYYY YYYY UV UV</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>YUV422_YU16     YYYY YYYY UUUU VVVV</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>YUV422_YV16     YYYY YYYY VVVV UUUU</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>YUV422_NV16     YYYY YYYY UVUV UVUV</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Components:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="checkBox_YUV">
         <property name="text">
          <string>YUV</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QCheckBox" name="checkBox_Y">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QCheckBox" name="checkBox_UV">
         <property name="text">
          <string>UV</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>CODED</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Filename:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_FileName"/>
   </item>
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_ImageSave</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_ImageSave</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
